tests/cases/conformance/forAwait/es6/forAwait.2.es6.ts(3,9): error TS1104: A 'for-await-of' statement is only allowed within an async function or async generator.


==== tests/cases/conformance/forAwait/es6/forAwait.2.es6.ts (1 errors) ====
    declare const ai: AsyncIterable<number>;
    function f() {
        for await (const x of ai) {
            ~~~~~
!!! error TS1104: A 'for-await-of' statement is only allowed within an async function or async generator.
        }
    }